### PR TITLE
Add support for `EnableStopPropagation` and `EnablePreventDefault`.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- For our current world Blazor both of these properties are always `true`. However, for other non-web Blazor scenarios the `EventHandler` attributes at that layer will turn off these features because they're specific to web.

#14517

### CODE REVIEW

File: src/Components/Components/src/EventHandlerAttribute.cs [hunk 1]
```diff
@@ -16,7 +16,9 @@ public sealed class EventHandlerAttribute : Attribute
         /// </summary>
         /// <param name="attributeName"></param>
         /// <param name="eventArgsType"></param>
-        public EventHandlerAttribute(string attributeName, Type eventArgsType)
```
Comment (pranavkm) [inline src/Components/Components/src/EventHandlerAttribute.cs]: You'll need to keep these around since this is a binary breaking change.

**Review by pranavkm — COMMENTED** [review]
Also need to update ref asm

**Comment by NTaylorMullen** [comment]
🆙 📅

**Comment by pranavkm** [comment]
Spoke to @NTaylorMullen offline. We can get this in as is, but he's going to file an issue to add some E2E tests for this.

**Review by pranavkm — APPROVED** [review]

**Pull request merged**